services/requester/key_store_component.go

```diff
@@ -0,0 +1,125 @@
+package requester
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/onflow/flow-evm-gateway/config"
+	"github.com/onflow/flow-go-sdk/access"
+	"github.com/onflow/flow-go-sdk/crypto"
+	"github.com/onflow/flow-go/module/component"
+	"github.com/onflow/flow-go/module/irrecoverable"
+	"github.com/rs/zerolog"
+)
+
+type KeyStoreComponent struct {
+	*KeyStore
+
+	log              zerolog.Logger
+	config           config.Config
+	client           access.Client
+	startupCompleted chan struct{}
+}
+
+var _ component.Component = (*KeyStoreComponent)(nil)
+
+func NewKeyStoreComponent(log zerolog.Logger, config config.Config, client access.Client) *KeyStoreComponent {
+	ks := &KeyStoreComponent{
+		log:              log,
+		config:           config,
+		client:           client,
+		startupCompleted: make(chan struct{}),
+	}
+
+	return ks
+}
+
+func (k *KeyStoreComponent) Start(ctx irrecoverable.SignalerContext) {
+	defer close(k.startupCompleted)
+
+	k.log.Info().Msg("starting key store component")
+
+	accountKeys := make([]*AccountKey, 0)
+	account, err := k.client.GetAccount(ctx, k.config.COAAddress)
+	if err != nil {
+		ctx.Throw(fmt.Errorf(
+			"failed to get signer info account for address: %s, with: %w",
+			k.config.COAAddress,
+			err,
+		))
+
+		return
+	}
+	signer, err := createSigner(ctx, k.config, k.log)
+
+	if err != nil {
+		ctx.Throw(err)
+
+		return
+	}
+	for _, key := range account.Keys {
+		accountKeys = append(accountKeys, &AccountKey{
```

Comment:
We are missing this piece of code: https://github.com/onflow/flow-evm-gateway/blob/main/bootstrap/bootstrap.go#L217-L221, from this loop here. This is quite important for the setup that we have now on `testnet` & `mainnet`.